src/client/src/orangehrmClaimPlugin/pages/ClaimEvent.vue

```diff
@@ -0,0 +1,288 @@
+        };
+      });
+    };
+
+    const http = new APIService(
+      window.appGlobal.baseUrl,
+      'api/v2/claim/events',
+    );
+    const {
+      showPaginator,
+      currentPage,
+      total,
+      pages,
+      pageSize,
+      response,
+      isLoading,
+      execQuery,
+    } = usePaginate(http, {
+      normalizer: claimEventDataNormalizer,
+      query: serializedFilters,
+    });
+    onSort(execQuery);
+
+    return {
+      http,
+      showPaginator,
+      currentPage,
+      isLoading,
+      total,
+      pages,
+      pageSize,
+      execQuery,
+      items: response,
+      response,
+      filters,
+      sortDefinition,
+    };
+  },
+  data() {
+    return {
+      headers: [
+        {
+          name: 'name',
+          title: this.$t('general.name'),
+          slot: 'title',
+          sortField: 'claimEvent.name',
+          style: {flex: 3},
+        },
+        {
+          name: 'status',
+          title: this.$t('general.status'),
+          sortField: 'claimEvent.status',
+          style: {flex: 2},
+        },
+        {
+          name: 'actions',
+          title: this.$t('general.actions'),
+          slot: 'action',
+          style: {flex: 1},
+          cellType: 'oxd-table-cell-actions',
+          cellConfig: {
+            delete: {
+              onClick: this.onClickDelete,
+              component: 'oxd-icon-button',
+              props: {
+                name: 'trash',
+              },
+            },
+            edit: {
+              onClick: this.onClickEdit,
+              props: {
+                name: 'pencil-fill',
+              },
+            },
+          },
+        },
+      ],
+      checkedItems: [],
+      ClaimEventStatuses: [
+        {id: 1, label: this.$t('general.active')},
+        {id: 0, label: this.$t('performance.inactive')},
+      ],
+    };
+  },
+
+  methods: {
+    async resetDataTable() {
+      this.checkedItems = [];
+      await this.execQuery();
+    },
+    async filterItems() {
+      await this.execQuery();
+    },
+    onClickReset() {
+      this.filters = {...defaultFilters};
+      this.filterItems();
+    },
+    onClickAdd() {
+      navigate('/claim/createEvents');
+    },
+    onClickDeleteSelected() {
+      const ids = [];
+      this.checkedItems.forEach((index) => {
+        ids.push(this.items?.data[index].id);
+      });
+      this.$refs.deleteDialog.showDialog().then((confirmation) => {
+        if (confirmation === 'ok') {
+          this.deleteItems(ids);
+        }
+      });
+    },
+    deleteItems(items) {
+      if (items instanceof Array) {
+        this.isLoading = true;
+        this.http
+          .deleteAll({
+            ids: items,
+          })
+          .then(() => {
+            return this.$toast.deleteSuccess();
+          })
+          .then(() => {
+            this.isLoading = false;
+            this.resetDataTable();
+          });
+      }
+    },
+    onClickDelete() {
```

Comment:
`onClickDelete` will pass the id of clicked row. don't use `checkedItems` for this